patches_for_WebRTC_org/m80/0007.4-Adding-video-profiles-capabilities-to-the-video-capt.patch

```diff
@@ -0,0 +1,1089 @@
+From e69d43b085062ce6d5ae2acb5f776c4533f81344 Mon Sep 17 00:00:00 2001
+From: Augusto Righetto <aurighet@microsoft.com>
+Date: Wed, 10 Jun 2020 17:01:45 -0700
+Subject: [PATCH] Adding video profiles capabilities to the video capture
+ module
+
+---
+ modules/video_capture/BUILD.gn                |   5 +
+ modules/video_capture/video_capture_defines.h |  33 +-
+ .../windows/device_info_winrt.cc              | 431 ++++++++++++++----
+ .../windows/help_functions_winrt.cc           | 187 +++++---
+ .../windows/help_functions_winrt.h            |  40 +-
+ .../windows/video_capture_winrt.cc            |  43 +-
+ 6 files changed, 563 insertions(+), 176 deletions(-)
+
+diff --git a/modules/video_capture/BUILD.gn b/modules/video_capture/BUILD.gn
+index 61b19993e0..67e75968b7 100644
+--- a/modules/video_capture/BUILD.gn
++++ b/modules/video_capture/BUILD.gn
+@@ -14,6 +14,11 @@ import("../../webrtc.gni")
+ # use the internal capturer.
+ rtc_library("video_capture_module") {
+   visibility = [ "*" ]
++
++  if (is_win && rtc_win_video_capture_winrt) {
```

Comment:
`rtc_win_video_capture_winrt` is always defined even in Win32, so will be `WEBRTC_VIDEO_CAPTURE_WINRT`, therefore I don't understand what the `#if WEBRTC_VIDEO_CAPTURE_WINRT` does, it is always true, no?

Reply:
It should always be true. Fixed.